### DISCUSSION TITLE
Make Open API Spec JSON location visible on the documentation

### DISCUSSION THREAD

**Discussion by michaelolof** [scalar/scalar#1998] [Ideas]
Hello.

First, Thank you for the awesome tool you've built and making it freely available.

Today you have a `Download API Spec` button for local downloads of the specification file.

That is great, but I see no way of getting the actual url of the JSON/YAML specification unless you view page source or ask the developer.

Can we make this visible on the documentation or change the download button to a link?

My use case is generating typescript types by pointing to the JSON/YAML specification

Thanks again.

**Comment by hanspagel** [comment]
Yes, you‘re right! I’ve created an issue: https://github.com/scalar/scalar/issues/2003
